README.md

```diff
@@ -94,25 +98,25 @@ make BUILD_TAGS='seccomp apparmor'
 | seccomp   | syscall filtering                  | libseccomp development library  |
 | selinux   | selinux process and mount labeling | <none>                          |
 | apparmor  | apparmor profile support           | libapparmor development library |
-### Validate Your cri Setup
+### Validate Your `cri` Setup
 A Kubernetes incubator project called [cri-tools](https://github.com/kubernetes-incubator/cri-tools)
-includes programs for exercising CRI implementations such as the `cri plugin`.
+includes programs for exercising CRI implementations such as the `cri` plugin.
 More importantly, cri-tools includes the program `critest` which is used for running
 [CRI Validation Testing](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-validation.md).
 
-Run the CRI Validation test to validate your installation of `containerd` with `cri`:
+Run the CRI Validation test to validate your installation of `containerd` with `cri` built-in:
 ```bash
 make test-cri
 ```
 ### Running a Kubernetes local cluster
 If you already have a working development environment for supported Kubernetes
 version, you can try `cri` in a local cluster:
 
-1. Start `containerd` as root in a first terminal:
+1. Start `containerd` with `cri` built-in as root in a first terminal:
```

Comment:
Running containerd is the same with or without cri built in. You might wasn't to say "Start the version of containerd with the cri plugin that you built and installed above:"

Reply:
Will do.

Reply:
Done